src/main/kotlin/build/buf/gradle/BreakingTask.kt

```diff
@@ -22,7 +22,6 @@ abstract class BreakingTask : DefaultTask() {
     fun bufBreaking() {
         execBuf(
             "breaking",
-            bufBuildPublicationFile,
```

Comment:
Curious why this is being removed?

Reply:
When using `breaking: ignore` with modules, there appears to be a limitation where the generated `.json` files contain file paths relative to the module dir, but the `ignore` clause itself contains paths relative to the `buf.yaml` file. The result is that comparing before vs. after doesn't handle the `ignore` properly when comparing `before.json` to `after.json`. However, comparing `before.json` to the actual repo contents does work correctly. So this change is just removing the reference to `after.json`

Reply:
I do like the strength of comparing the built images, but if this is equivalent *always*, then I guess it doesn't matter. 

We could also change the default format from JSON if that solves the problem.